<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditorClass</class>
 <widget class="QMainWindow" name="TextEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_Preview"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <widget class="QMenu" name="menuFont">
     <property name="title">
      <string>Font</string>
     </property>
     <addaction name="actionBold"/>
     <addaction name="actionItalic"/>
     <addaction name="actionUnderline"/>
    </widget>
    <addaction name="menuFont"/>
    <addaction name="actionColor"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="File">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPrint"/>
  </widget>
  <widget class="QToolBar" name="Edit">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
  </widget>
  <widget class="QToolBar" name="Format">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="actionColor"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/new.png</normaloff>:/TextEditor/Resources/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>Create a new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/open.png</normaloff>:/TextEditor/Resources/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/save.png</normaloff>:/TextEditor/Resources/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save the file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="statusTip">
    <string>Save file as ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/print.png</normaloff>:/TextEditor/Resources/icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="statusTip">
    <string>Print the file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/printPreview.png</normaloff>:/TextEditor/Resources/icons/printPreview.png</iconset>
   </property>
   <property name="text">
    <string>Print Preview</string>
   </property>
   <property name="statusTip">
    <string>Preview the file </string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/exit.png</normaloff>:/TextEditor/Resources/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/undo.png</normaloff>:/TextEditor/Resources/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="statusTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/redo.png</normaloff>:/TextEditor/Resources/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="statusTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/cut.png</normaloff>:/TextEditor/Resources/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="statusTip">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/copy.png</normaloff>:/TextEditor/Resources/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/paste.png</normaloff>:/TextEditor/Resources/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/color.png</normaloff>:/TextEditor/Resources/icons/color.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="statusTip">
    <string>Color</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/bold.png</normaloff>:/TextEditor/Resources/icons/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="statusTip">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/italic.png</normaloff>:/TextEditor/Resources/icons/italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="statusTip">
    <string>Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/underline.png</normaloff>:/TextEditor/Resources/icons/underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="statusTip">
    <string>Underline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="TextEditor.qrc">
     <normaloff>:/TextEditor/Resources/icons/aboutQt.png</normaloff>:/TextEditor/Resources/icons/aboutQt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="statusTip">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TextEditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>textEdit</sender>
   <signal>textChanged()</signal>
   <receiver>TextEditorClass</receiver>
   <slot>textChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeTitle()</slot>
  <slot>textChanged()</slot>
 </slots>
</ui>
